src/components/Mathfield.tsx

```diff
@@ -1,13 +1,13 @@
 import { useEffect, useRef } from 'react'
 import { MathfieldElement } from 'mathlive'
 
-MathfieldElement.fontsDirectory = ''
+MathfieldElement.fontsDirectory = 'https://unpkg.com/mathlive/dist/fonts/'
 MathfieldElement.soundsDirectory = null
 window.mathVirtualKeyboard.layouts = {
   rows: [
     ['1', '2', '3', '4', '5', '6', '7', '8', '9', '0'],
     [
-      '\\sqrt{#0}', '#@^{#?}', '+', '-', '\\times', '\\frac{#@}{#?}', '|#0|', '(', ')'
+      '\\sqrt{#0}', '\\sqrt[#1]{#0}', '#@^{#?}', '+', '-', '\\times', '\\frac{#@}{#?}', '|#0|', '(', ')'
```

Comment:
I know @MReyna12 and I discussed this a bit in a previous PR, but I'm having trouble understanding [this documentation](https://cortexjs.io/mathlive/guides/virtual-keyboards/#placeholder-tokens) as it relates to the differences between things like:
* `'\\sqrt[#1]{#0}'`
* `'\\sqrt[#@]{#?}'`
* ...

I see in the keyboard the latter creates a blue square vs an empty one, but experimentally I can't seem to figure out the implication from a user perspective. I wanted to check if you may have any thoughts / ideas (otherwise we can leave it as-is for now and revisit if we ever find an issue with it :man_shrugging: )